src/thunderbots/software/util/canvas_messenger/canvas_messenger.cpp

```diff
@@ -0,0 +1,113 @@
+#include "canvas_messenger.h"
+
+#include "util/constants.h"
+
+namespace Util
+{
+    std::shared_ptr<CanvasMessenger> CanvasMessenger::getInstance()
+    {
+        static std::shared_ptr<CanvasMessenger> vm(new CanvasMessenger);
+        return vm;
```

Comment:
could we change `vm` to `canvas_messenger` just to keep them in sync?

Reply:
Changed